packages/clay-drilldown/stories/index.tsx

```diff
@@ -0,0 +1,39 @@
+/**
+ * SPDX-FileCopyrightText: © 2020 Liferay, Inc. <https://liferay.com>
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import '@clayui/css/lib/css/atlas.css';
+const spritemap = require('@clayui/css/lib/images/icons/icons.svg');
+import {storiesOf} from '@storybook/react';
+import React from 'react';
+
+import ClayDrilldown from '../src';
+
+storiesOf('Components|ClayDrilldown', module).add('default', () => (
+	<ClayDrilldown
+		className="dropdown-menu show"
+		items={[
+			{href: '#', title: 'One'},
+			{onClick: () => alert('test'), title: 'Two'},
+			{
+				items: [
+					{href: '#', title: 'Four'},
+					{
+						items: [{title: 'Six'}, {title: 'Seven'}],
+						title: 'Five',
+					},
+				],
+				title: 'Three',
+			},
+		]}
```

Comment:
I like this style of API, but I am concerned with the depth of the API and if it could be bad for use in the case of DXP, for example Some of the data is handled in the backend and then sent to the component.

Thinking if we can only have a flat list of items, something like this:

```js
const items = {
	'x0a3': [
		{href: '#', title: 'One'},
		{onClick: () => alert('test'), title: 'Two'},
		{title: 'Three', children: 'x0a4'},
	],
	'x0a4': [
		{href: '#', title: 'Four'},
		{title: 'Five', children: 'x0a5'}
	],
	'x0a5': [
		{title: 'Six'},
		{title: 'Seven'}
	]
};
```

Just adding a new API option! What do you think? I'm fine with going on like this